src/index.html

```diff
@@ -1,13 +1,70 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page">
+    <div class="product-card page__product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__pic"
+        src="./images/imac.jpeg"
+        alt="Моноблок APPLE A1419 iMac 27' Retina 5K (MNED2UA/A)"
+      >
+
+      <h2 class="product-card__name">
```

Comment:
Maybe, we must use tags h2 ,h3, h4 etc. , if tag h1 is on page

Reply:
hm, in the checklist they wrote:
[SEMANTICS] - Use semantic tags like h2, p ...
so I followed instructions

perhaps we will have h1 tag in the future...